payroll-processor-client/src/app/employee/employee-list/employee-list.component.ts

```diff
@@ -11,20 +15,23 @@ import { faSmileBeam, faSkull } from '@fortawesome/free-solid-svg-icons';
 export class EmployeeListComponent implements OnInit, OnDestroy {
   faSkull = faSkull;
   faSmileBeam = faSmileBeam;
+  employees = of<Employee[]>();
 
-  constructor(private dataService: DataService) {}
+  constructor(
+    private employeeListService: EmployeeListService,
```

Comment:
We can mark these as `readonly` as well:

`private readonly employeeListService: EmployeeListService`.

We can also simplify the names since there are only 2 types being injected:

```typescript
private readonly service: EmployeeListService,
private readonly query: EmployeListQuery
```

Reply:
fixed!